src/Microsoft.AspNet.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -17,6 +18,9 @@ public class DateHeaderValueManager : IDisposable
         private readonly TimeSpan _timerInterval;
 
         private volatile string _dateValue;
+        private volatile int _activeDateBytes;
```

Comment:
I'm not a huge fan of the bit twiddling. Could we make this a bool and use `_activeDateBytes = !_activeDateBytes` instead of `_activeDateBytes ^= 1`?